app/src/main/java/org/simple/clinic/home/patients/PatientsScreen.kt

```diff
@@ -166,4 +169,8 @@ open class PatientsScreen(context: Context, attrs: AttributeSet) : RelativeLayou
   fun openEnterCodeManuallyScreen() {
     screenRouter.push(EnterOtpScreenKey())
   }
+
+  fun showScanCardButton(scanCardButtonEnabled: Boolean) {
```

Comment:
`showScanCardButton()` suggests that this function will always show the button. Let's do `setScanCardButtonVisible()` instead? Then the param could be renamed to just `enabled`.

Reply:
Done.